papaparse.js

```diff
@@ -839,6 +848,107 @@
 	ReadableStreamStreamer.prototype.constructor = ReadableStreamStreamer;
 
 
+	function DuplexStreamStreamer(_config) {
+		var Duplex = require('stream').Duplex;
```

Comment:
@pokoli I just want to draw attention to the fact that this exists.  This is the first external dependency that this library uses.  I don't think it will be an issue because I have buried inside this constructor that SHOULD only be invoked when ran in a node environment.

Reply:
Mmmm... Is not possble to avoid an external dependency? I've review the node api docs and the pipe functions are part of the core solution since long time ago. 

I will prefer to minimize this kind of dependencies.

Reply:
I totally agree with you.  The only way I know off the top of my mind would be to literally copy the code from node's core library into this library.  Duplex is more than just an interface as it handles a lot of the heavy lifting to make streams possible.
Here is the code for duplex which also depends on `stream.Writable` and `stream.Readable`
https://github.com/nodejs/node/blob/master/lib/_stream_duplex.js

Reply:
Ok, I misunderstood your comment. I thought this was an external library that was required to be installed from npm. If it's part of node source code it's ok to depend on it as it's a core library supported by node.

The current design is safe enough to not fail when running outside of node. So it's ok for me. That's what I understood on the first review and that's why I didn't say anything about it ;)